src/Tribe/Commerce/Warnings.php

```diff
@@ -39,6 +40,24 @@ public function recurring_event_warning_msg( $post_id ) {
 		$this->show_notice( $warning );
 	}
 
+	/**
+	 * Add Provider missing warning for tickets.
+	 *
+	 * @param $post_id
+	 */
+	public function add_commerce_provider_warning( $post_id ) {
+		$available_modules = array_diff_key( Tribe__Tickets__Tickets::modules(), [ 'Tribe__Tickets__RSVP' => true ] );
+
+		if ( count( $available_modules ) > 0 ) {
+			return;
+		}
+
+		$kb_link = 'http://m.tri.be/1ao5';
+		$message = sprintf( __( 'There is no payment gateway configured. To create tickets, you\'ll need to enable and configure an ecommerce solution. <a href="%1$s" target="_blank" rel="noopener noreferrer">[Learn more]</a>', 'event-tickets' ), $kb_link );
+
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: cc85894db7d445bd10d3f1316f7262b5</sub>